epifor/data/csse.py

```diff
@@ -23,27 +24,61 @@
 class CSSEData:
     def __init__(self):
         self.df = None
-
-    def load(self, pattern):
+        self.HIST_FILE_NAME = "csse_history_data.h5"
```

Comment:
this could be therefore a class attribute directly, given it's a constant, no?

Reply:
ok